assets/app/scripts/controllers/buildConfig.js

```diff
@@ -149,8 +82,43 @@ angular.module('openshiftConsole')
           }
         );
 
+      watches.push(DataService.watch("builds", context, function(builds, action, build) {
+        $scope.emptyMessage = "No builds to show";
+        // TODO we should send the ?labelSelector=buildconfig=<name> on the API request
```

Comment:
can you actually take this comment out, it should have been removed as part of @benjaminapetersen PR that added the selector but i missed it